src/components/filters/DateFilter/utils/tests/FormattingUtils.spec.ts

```diff
@@ -0,0 +1,24 @@
+// (C) 2019 GoodData Corporation
+import { sanitizeLocaleForMoment } from "../FormattingUtils";
+
+describe("sanitizeLocaleForMoment", () => {
+    it("should sanitize the zh-Hans locale", () => {
+        const expected = "zh-CN";
+        const actual = sanitizeLocaleForMoment("zh-Hans");
+        expect(actual).toEqual(expected);
+    });
+
+    it.each([["de-DE"], ["en-US"], ["es-ES"], ["fr-FR"], ["ja-JP"], ["nl-NL"], ["pt-BR"], ["pt-PT"]])(
```

Comment:
If each item contains only one member, it can be simplified to
```suggestion
    it.each(["de-DE", "en-US", "es-ES", "fr-FR", "ja-JP", "nl-NL", "pt-BR", "pt-PT"])(
```